build/yamls/windows/containerd-with-ovs/containerd-with-ovs.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    component: antrea-agent
```

Comment:
Since it uses patchesStrategicMerge, does it need this `labels` and the following `selector` and `template.metadata`?

Reply:
removed, Thanks!